src/plugins/http_params_hook.py

```diff
@@ -0,0 +1,75 @@
+import requests
+from airflow.hooks.http_hook import HttpHook
+
+
+class HttpParamsHook(HttpHook):
+    """ Unfortunately, the default HttpHook makes it impossible
+        to combine query params with a payload in the body, so
+        we need a subclass with a slightly modified copy of the
+        run method
+    """
+
+    def run(
+        self,
+        endpoint,
+        params=None,
+        data=None,
+        headers=None,
+        extra_options=None,
+        **request_kwargs
+    ):
+        r"""
+        Performs the request
+
+        :param endpoint: the endpoint to be called i.e. resource/v1/query?
+        :type endpoint: str
```

Comment:
types niet als annotatie doen?

Reply:
Dat doet Airflow nog niet, wschl. om nog compatibel te blijven met oudere pythons. Maar, dat is voor eigenlijk geen issue. En dan is het mooier om ze wel te gebruiken, ik ga ze toevoegen.